docsets/OTRS/docset.json

```diff
@@ -12,32 +12,64 @@
         "_comment": ""
     },
     {
-        "version": "5.0.8",
-        "archive": "versions/5.0/otrs5.tgz",
+        "version": "5.0.42",
+        "archive": "versions/5.0/otrs5.tgz"
     },
     {
-        "version": "4.0.16",
-        "archive": "versions/4.0/otrs4.tgz",
+        "version": "4.0.33",
+        "archive": "versions/4.0/otrs4.tgz"
     },
     {
-        "version": "3.3.15",
-        "archive": "versions/3.3/otrs33.tgz",
+        "version": "3.3.20",
+        "archive": "versions/3.3/otrs33.tgz"
     },
     {
         "version": "3.2",
-        "archive": "versions/3.2/otrs32.tgz",
+        "archive": "versions/3.2/otrs32.tgz"
     },
     {
         "version": "3.1",
-        "archive": "versions/3.1/otrs31.tgz",
+        "archive": "versions/3.1/otrs31.tgz"
     },
     {
         "version": "3.0",
-        "archive": "versions/3.0/otrs30.tgz",
+        "archive": "versions/3.0/otrs30.tgz"
     },
     {
         "version": "2.4",
-        "archive": "versions/2.4/otrs24.tgz",
+        "archive": "versions/2.4/otrs24.tgz"
     },
+    {
+        "version":"FAQ",
+        "archive": "versions/FAQ.tgz"
```

Comment:
This path is wrong. It should be `versions/FAQ/FAQ.tgz`. Paths below are wrong as well.

Reply:
Thanks! I missed that. It is fixed with the latest commit